scripts/validate-packages.py

```diff
@@ -133,6 +138,22 @@ def _validate_revision(given_package, revision, path):
                  'only supported when minDcosReleaseVersion is greater than '
                  '1.8.')
 
+def _validate_resource_assets(resource_json):
+    all_good = True
+    for (type, uri) in resource_json.get('assets',{}).get('uris',{}).items():
+
+        # Check for non-https assets
+        if (urlparse(uri).scheme != 'https'):
+            all_good = False
+            eprint("\n\t\t\tWARNING non-https link: {}".format(uri), end='')
+
+        # check for links on downloads downloads.mesosphere.com that do not point
+        # to a dedicated sub dir.
+        if (urlparse(uri).hostname == 'downloads.mesosphere.com' and not urlparse(uri).path.startswith('/universe/assets/') ):
+            all_good = False
+            eprint("\n\t\t\tWARNING downloads.mesosphere.com asset in stray directory: {}".format(uri), end='')
+
+    return all_good
```

Comment:
I was planing on using these warnings in the first step, then going through the packages myself and correct these for each package with a new revision. This is supposed to be a soft launch of a few rules:
* only https links
* consolidated path to packages for easier replication of a whole universe

After that we should make it hard errors. wdyt?

Also, whats your opinion on my comment above that this is very mesosphere specific and maybe should go in the other checker repository?